spring-boot-unit-testing/src/test/java/dev/muano/springbootunittesting/repository/PokemonRepoTests.java

```diff
@@ -32,6 +35,45 @@ public void PokemonRepo_SaveAll_ReturnSavedPokemon() {
         Assertions.assertThat(pokemon).isNotNull();
         Assertions.assertThat(pokemon.getId()).isGreaterThan(0);
     }
+
+    @Test
+    public void PokemonRepo_GetAll_ReturnMoreThanOnePokemon() {
+
+        //Arrange
+        Pokemon pokemon = Pokemon.builder()
+                .name("pikachu")
+                .type("electric")
+                .build();
+
+        Pokemon pokemon2 = Pokemon.builder()
```

Comment:
Here there is a duplicated Pokemon

Reply:
It is just a dummy pokemon so it shouldn't matter since we are not testing for redundecies